timescale-container/quantumleap-db-setup.py

```diff
@@ -205,7 +205,7 @@ class CreateDb:
     OWNER ${db_user}
     ENCODING 'UTF8';
 
-\connect ${db_name}
+\\connect ${db_name}
```

Comment:
why this change? this may break the db setup, correct @c0c0n3 ?

Reply:
yep, good catch! @pooja1pathak can you confirm you didn't do this yourself? I think it was autopep. I'm busy opening an issue about it. In the meantime, @pooja1pathak  if you could please undo this change, i.e. the deleted line is correct.

Reply:
@chicco785 @pooja1pathak see #618

Reply:
> yep, good catch! @pooja1pathak can you confirm you didn't do this yourself? I think it was autopep. I'm busy opening an issue about it. In the meantime, @pooja1pathak if you could please undo this change, i.e. the deleted line is correct.

No, it was autopep. I have reverted the change.